src/faq.mdx

```diff
@@ -1,5 +1,70 @@
 *Note: This article is written by past USACO Finalists and is not an official FAQ.*
 
-## Q: What is USACO?
+## Q: What language should I use for USACO?
 
-lorem ipsum dolor sit amet
\ No newline at end of file
+In general, we recommend the following:
+
+- **If you already know one of these languages, just use it.**
+  - If you know multiple languages, we recommend you pick C++ over Java, and Java over Python.
+- **For Bronze, any language will do.** It is possible to receive full credit with C++, Java, and Python in Bronze.
```

Comment:
second part kinda redundant??

Reply:
I think its worth mentioning that all languages can pass in Bronze (but not silver/gold/plat)

Reply:
sry i meant the second part of **For Bronze, any language will do.** It is possible to receive full credit with C++, Java, and Python in Bronze. this sentence

Reply:
yeah I think that second part is worth keeping (because you can't get full credit all the time in Silver). "any language will do" doesn't necessarily imply that all languages will pass right? It suggests that you can pick any language, not that any language will get full credit.

(If a lot of other people think it's redundant I'm fine w/ getting rid of it though)